### ISSUE TITLE
LSP (yamlls) compatible with yaml-mode?

### ISSUE THREAD

**Issue by LosAlamosAl** [yoshiki/yaml-mode#100]
A year-old [thread on Reddit](https://www.reddit.com/r/emacs/comments/o3jf2y/lsp_and_yaml/) says that (at that time) only `yaml-language-server@0.18.0` was compatible with `yaml-mode`. I'm currently (given that it's been a year) trying `yaml-language-server@1.10.0` and having load of problems. Before I commence debugging can you spell out the current lay of the land WRT `yamlls` compatibility?

**Comment by wasamasa** [comment]
No, I can't. I don't use LSP at all and do not really see the point of using it with YAML, since it's not a programming language.
